metadata-ingestion/tests/unit/utilities/test_utilities.py

```diff
@@ -1,302 +0,0 @@
-    GROUP BY
-        1,
-        2,
-        3,
-        4,
-        5)
-"""
-
-    columns_list = SqlLineageSQLParser(sql_query).get_columns()
-    columns_list.sort()
-    assert columns_list == ["c", "date", "e", "u", "x"]
-
-
-def test_sqllineage_sql_parser_get_tables_from_templated_query():
-    sql_query = """
-        SELECT
-          country,
-          city,
-          timestamp,
-          measurement
-        FROM
-          ${my_view.SQL_TABLE_NAME} AS my_view
-"""
-    tables_list = SqlLineageSQLParser(sql_query).get_tables()
-    tables_list.sort()
-    assert tables_list == ["my_view.SQL_TABLE_NAME"]
-
-
-def test_sqllineage_sql_parser_get_columns_from_templated_query():
-    sql_query = """
-        SELECT
-          country,
-          city,
-          timestamp,
-          measurement
-        FROM
-          ${my_view.SQL_TABLE_NAME} AS my_view
-"""
-    columns_list = SqlLineageSQLParser(sql_query).get_columns()
-    columns_list.sort()
-    assert columns_list == ["city", "country", "measurement", "timestamp"]
-
-
-def test_sqllineage_sql_parser_with_weird_lookml_query():
-    sql_query = """
-    SELECT date DATE,
-             platform VARCHAR(20) AS aliased_platform,
-             country VARCHAR(20) FROM fragment_derived_view'
-    """
-    columns_list = SqlLineageSQLParser(sql_query).get_columns()
-    columns_list.sort()
-    assert columns_list == ["aliased_platform", "country", "date"]
-
-
-def test_sqllineage_sql_parser_tables_from_redash_query():
-    sql_query = """SELECT
-name,
-SUM(quantity * list_price * (1 - discount)) AS total,
-YEAR(order_date) as order_year
-FROM
-`orders` o
-INNER JOIN `order_items` i ON i.order_id = o.order_id
-INNER JOIN `staffs` s ON s.staff_id = o.staff_id
-GROUP BY
-name,
-year(order_date)"""
-    table_list = SqlLineageSQLParser(sql_query).get_tables()
-    table_list.sort()
-    assert table_list == ["order_items", "orders", "staffs"]
-
-
-def test_sqllineage_sql_parser_tables_with_special_names():
-    # The hyphen appears after the special token in tables names, and before the special token in the column names.
-    sql_query = """
-SELECT `column-date`, `column-hour`, `column-timestamp`, `column-data`, `column-admin`
-FROM `date-table` d
-JOIN `hour-table` h on d.`column-date`= h.`column-hour`
-JOIN `timestamp-table` t on d.`column-date` = t.`column-timestamp`
-JOIN `data-table` da on d.`column-date` = da.`column-data`
-JOIN `admin-table` a on d.`column-date` = a.`column-admin`
-"""
-    expected_tables = [
-        "admin-table",
-        "data-table",
-        "date-table",
-        "hour-table",
-        "timestamp-table",
-    ]
-    expected_columns = [
-        "column-admin",
-        "column-data",
-        "column-date",
-        "column-hour",
-        "column-timestamp",
-    ]
-    assert sorted(SqlLineageSQLParser(sql_query).get_tables()) == expected_tables
-    assert sorted(SqlLineageSQLParser(sql_query).get_columns()) == expected_columns
```

Comment:
let's keep these tests around, and make it test with the new parser

see https://github.com/datahub-project/datahub/blob/b8144699fdb37c3e935bef92b628d1d956e3f666/metadata-ingestion/tests/unit/bigquery/test_bigquery_sql_lineage.py#L7-L8 for an example

Reply:
Hi @hsheth2, 

I've updated the tests to align with SQLGlot, and I noticed some changes in the outputs. Could you please verify them?